deploy/examples/cluster.yaml

```diff
@@ -281,6 +281,16 @@ spec:
     # No values or 0 means that the operator will wait until the placement groups are healthy before unblocking the next drain.
     pgHealthCheckTimeout: 0
 
+  # csiDriverOptions defines defines CSI Driver settings applied per cluster.
+  csiDriverOptions:
```

Comment:
how abou the below CR template

```code
csi:
   general:
        x: x
   cephfs:
        x:x
   rbd:
      x:x
   nfs:
     x:x
```

The above might helps us to keep the config specific and easily extendable?

Reply:
@Madhu-1 Can you please check it now ? 
I've segregated settings into common and cephfs for now.